src/index.html

```diff
@@ -4,9 +4,397 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header 
+      class="header"
+    >
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo-image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#Apple"
+              class="is-active"
+            >
```

Comment:
```suggestion
            <a href="#Apple" class="is-active">
```
Same everywhere where less that 3 attributes